components/navigation/Filter.js

```diff
@@ -0,0 +1,38 @@
+import {useState} from 'react';
+import PropTypes from 'prop-types';
+
+export default function Filter({data, onFilter}) {
+    const [state, setState] = useState('Add filter');
```

Comment:
when I click on Add filter and select one, the drop-down is not collapsed, it is because the intention is to have multiple selections there?

Reply:
No. I think something went wrong there. Let me fix that and update the PR

Reply:
@derberg I think it has something to do with Tailwind CSS limitations, I'll use a custom CSS